src/js/components/atoms/text/copy/copy.scss

```diff
@@ -1,5 +1,5 @@
 // moved original scss from src/scss/atomic/atoms/_type.scss to here
-p {
+p, ul, ol {
```

Comment:
can you move the ul ol to lists.scss?

Reply:
Im leaving it in copy, lists.scss has al kinds of weird other styling